sql/char_fishing.sql

```diff
@@ -0,0 +1,22 @@
+-- --------------------------------------------------------
+-- Add history for player fishing data
+-- --------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS `char_fishing` (
+    -- Fishing Catch History
+    `charid`           int(10) unsigned NOT NULL,
+    `fish_list`        blob,
+    `fish_linescast`   int(6)  unsigned NOT NULL DEFAULT 0,
+    `fish_reeled`      int(5)  unsigned NOT NULL DEFAULT 0,
+    `fish_longest`     int(4)  unsigned NOT NULL DEFAULT 0,
+    `fish_longest_id`  int(4)  unsigned NOT NULL DEFAULT 0,
+    `fish_heaviest`    int(4)  unsigned NOT NULL DEFAULT 0,
+    `fish_heaviest_id` int(4)  unsigned NOT NULL DEFAULT 0,
```

Comment:
Out of interest, why are the display widths for these all a bit different?

Reply:
I feel like I may have typoed some of those.  I intended for them to be roughly in line with what would make sense (would someone actually cast 1 million lines?  Maybe...) but then the IDs should be 5-wide and not 4.  I can adjust these to be standardized, but not sure if there's a preferred way of having them done.

Reply:
For info, I've removed these entries in the upcoming push as they will be separate when I add the fishing history